libs/langgraph/tests/test_pregel_async.py

```diff
@@ -6006,6 +6006,381 @@ async def side(state: State):
     assert times_called == 1
 
 
+@pytest.mark.parametrize(
+    "checkpointer",
+    [
+        MemorySaverAssertImmutable(),
+        AsyncSqliteSaver.from_conn_string(":memory:"),
+    ],
+    ids=[
+        "memory",
+        "sqlite",
+    ],
+)
+async def test_nested_graph_interrupts(checkpointer: BaseCheckpointSaver) -> None:
```

Comment:
whenever using sqlite saver you need to close it at end of test, see other tests